src/main/java/org/sonar/scm/git/blame/FileBlamer.java

```diff
@@ -78,21 +78,22 @@ public void initialize(ObjectReader objectReader, StatefulCommit commit) {
 
   /**
    * Blame all remaining regions to the commit
+   *
+   * @param source - commit that will be used to associate blame data with remaining regions in files
    */
-  public void processResult(StatefulCommit source) {
-
+  public void saveBlameDataForFiles(StatefulCommit source) {
     String commitHash = source.getCommit().getName();
     String authorEmail = source.getCommit().getAuthorIdent().getEmailAddress();
     Date commitDate = source.getCommit().getCommitterIdent().getWhen();
     for (FileCandidate sourceFile : source.getAllFiles()) {
       if (sourceFile.getRegionList() != null) {
-        blameResult.process(commitHash, commitDate, authorEmail, sourceFile);
+        blameResult.storeBlameDataForFile(commitHash, commitDate, authorEmail, sourceFile);
```

Comment:
Why do we use both save and store ? To me, "store" refers more to a fileSystem for instance, but I may be wrong

Reply:
fixed.